index.html

```diff
@@ -1,9 +1,103 @@
-Create an Org and get more out of the tools your team already knows and loves. Zero configuration Create an org, add
-your team, and start collaborating. Nothing to configure, set up, or manage. Team management Control who has access to
-what modules within your team namespace using straightforward team management capabilities. Familiar features npm Orgs
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css" />
+
+</head>
+
+<body>
+    <header>
+        <div class="topheader">
+            <div>
+                <img class="blackHeart" src="images/black-heart.png" alt="black heart" />
+                <span> Nifty Penguin Magic </span>
+            </div>
+            <nav class="navbar">
+                <ul>
+                    <li><a href="#"> npm Enterprise </a></li>
+                    <li><a href="#"> Products </a></li>
+                    <li><a href="#"> Solutions </a></li>
+                    <li><a href="#"> Resources </a></li>
+                    <li><a href="#"> Docs </a></li>
+                    <li><a href="#"> Support </a></li>
+                </ul>
+            </nav>
+        </div>
+
+        <div class="searchPackages">
+            <img class="npmLogo" src="images/npm-logo.png" alt="npm-logo" width=45 />
+            <form action="/action_page.php">
+                <img class="magnglass" src="images/magnifying-glass.png" alt="search" width=20 />
+                <input class="search" type="search" placeholder="search" aria-label="search" >
+            </form>
+            <button type="button" class="button">Search</button>
+
+            <a class="join" href="#Join">Join</a>
+            <a class= "login" href="#Login">Log in</a>
+
+            <!-- add logo, search bar and buttons (or links and style them as buttons later) -->
+        </div>
+    </header>
+    <section id="firstSection">
+        <h8 id="amazing">Build amazing things</h8>
+        <p id="firstParagraph">Essential JavaScript development tools that help you go to market faster and build powerful applications
+            using modern open source code.</p>
+<div class="planjoin">
+    <button type="SeePlans" class="SeePlans">See Plans</button>
+    <button type="JoinForFree" class="JoinForFree">Join for free</button>
+
+</div>
+    </section>
+
+
+    <section class="secondSection">
+        <img class="hexagon" src="images/triangle-hexagon.svg" alt="hexagon" />
+        <h7>Bring the best open source to your company</h7>
+        <p5>npm is the tool used by over 11,000,000 JavaScript developers around the world. Your developers already use
```

Comment:
For paragraphs, you can use a `<p>` element. You should update line 60, 69, 75, 81 and 87 accordingly